### PR TITLE
feat(core): organization app role apis

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

- create organization - application - organization role relations table
- add apis for handling organization - application - organization role relations
- add `PUT` api for replacing all organization role relations for a organization - user pair

## Testing

integration tests added

## Checklist

- [ ] `.changeset` (not this one)
- [ ] unit tests
- [x] integration tests
- [x] necessary TSDoc comments

### CODE REVIEW

File: packages/core/src/routes/organization/index.user-role-relations.ts
Comment (gao-sun) [inline packages/core/src/routes/organization/index.user-role-relations.ts:1]: only changed the order of routes for alignment

File: packages/core/src/routes/organization/index.ts
Comment (gao-sun) [inline packages/core/src/routes/organization/index.ts:1]: only changed the order of routes (with tiny comment addition) for alignment

File: packages/core/src/queries/organization/role-user-relations.ts
Comment (gao-sun) [inline packages/core/src/queries/organization/role-user-relations.ts:1]: only order changes

File: packages/core/src/routes/organization/index.user-role-relations.ts
Comment (gao-sun) [inline packages/core/src/routes/organization/index.user-role-relations.ts:1]: only changed the order of routes (with tiny comment addition) for alignment

File: packages/core/src/queries/organization/index.ts [hunk 1]
```diff
@@ -283,6 +284,7 @@ export default class OrganizationQueries extends SchemaQueries<
     ),
     /** Queries for organization - user relations. */
     users: new UserRelationQueries(this.pool),
+    // TODO: Rename to `usersRoles`
```
Comment (gao-sun) [inline packages/core/src/queries/organization/index.ts:287]: updated in #6072

File: packages/core/src/routes/organization/index.application-role-relations.ts
Comment (wangsijie) [inline packages/core/src/routes/organization/index.application-role-relations.ts:1]: Noticed this file name format in recent PRs, is this a new convension?

**Review by wangsijie — APPROVED** [review]

File: packages/core/src/routes/organization/index.application-role-relations.ts
Comment (gao-sun) [reply packages/core/src/routes/organization/index.application-role-relations.ts:1]: not really. actually other routes like organization invitations should be put in a separate folder. i'll update it soon

Comment (gao-sun) [reply packages/core/src/routes/organization/index.application-role-relations.ts:1]: #6078

**Pull request merged**